Update how-to-connect-emergency-ad-fs-certificate-rotation.md (#1695)
MicrosoftDocs/entra-docs: docs/identity/hybrid/connect/how-to-connect-emergency-ad-fs-certificate-rotation.md

## Determine your Token Signing Certificate thumbprint

To revoke the old Token Signing Certificate that AD FS is currently using, you need to determine the thumbprint of the token-signing certificate. From your ADFS Server do the following:

> 1. Connect to the Microsoft Entra Module by running in PowerShell:

Reviewer: ```suggestion
1. Connect to the Microsoft Entra PowerShell module:
```

   `Connect-Entra -Scopes 'Domain.Read.All'`.

1. Document both your on-premises and cloud Token Signing Certificate thumbprint and expiration dates by running:
  
   *`Get-AdfsCertificate -CertificateType token-signing>`

   *`Get-EntraFederationProperty -DomainName <your_domain.com> | FL Source, SigningCertificate`.
  

> 1. Copy down the thumbprint. You'll use it later to remove the existing certificates.

Reviewer: ```suggestion
1. Copy the thumbprint. You'll use it later to remove the existing certificates.
```

## Update Microsoft Entra ID with the new token-signing certificate

1. Connect to Microsoft Entra ID by running the following command: 

   `Connect-Entra -Scopes 'Domain.Read.All'`
   
1. Enter your [Hybrid Identity Administrator](/entra/identity/role-based-access-control/permissions-reference#hybrid-identity-administrator) credentials.

1. Optionally, verify whether an update is required by checking the current certificate information in Microsoft Entra ID. To do so, run the following command:
  
   `Get-EntraFederationProperty -DomainName <your_domain.com> | FL Source, SigningCertificate` and convert the Base64 Encoded cert to a readble format to check the certificate expiration and thumbprint. 

1. To update the certificate information in Microsoft Entra ID, run the following command: `Update-MgDomainFederationConfiguration -DomainId <your_domain.com> -InternalDomainFederationId <hex_domainID>`.

   >[!IMPORTANT]

>    You can get the **-InternalDomainFederationId** value by running the commando below:

Reviewer: ```suggestion
   You can get the **-InternalDomainFederationId** value by running the following command:
```

* `Get-EntraFederationProperty -DomainName your_domain.com`

>    <img width="1774" height="165" alt="Get-EntraFedProperty" src="https://github.com/user-attachments/assets/5eac3a9f-a1e8-49e7-b3fb-36360a2ee211" />

Reviewer: We'll need to add this image via the private repo. I'll help with that.
